docs/management/admin/trusted-apps.asciidoc

```diff
@@ -16,27 +16,30 @@ To add a trusted application:
 
 . Click *Add trusted application*.
 
-. Fill in the following fields in the *Add trusted application* pane:
+. Fill in the following fields in the *Add trusted application* flyout:
 
 * `Name your trusted application`: Enter a name for the trusted application.
 
 * `Description`(Optional): Enter a description for the trusted application.
 
 * `Select operating system`: Select the appropriate operating system from the drop-down.
 
-* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you're adding a trusted application on Windows) `Signature`.
-+
-NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize visibility gaps in the {security-app}, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.
+* `Field`: Select the field you want to use to identify the trusted application:
+** `Hash`: The hash value of the application's executable. The hash can be MD5, SHA-1, or SHA-256.
```

Comment:
```suggestion
** `Hash`: The application's hash value. Either MD5, SHA-1, or SHA-256.
```

Reply:
I think it's important to specify the executable, to avoid any confusion with other application files, but in general I like the edits for brevity.
```suggestion
** `Hash`: The hash value of the application's executable. Either MD5, SHA-1, or SHA-256.
```